components/layout_2020/flow/inline.rs

```diff
@@ -2407,3 +2410,27 @@ fn char_prevents_soft_wrap_opportunity_when_before_or_after_atomic(character: ch
         class == XI_LINE_BREAKING_CLASS_WJ ||
         class == XI_LINE_BREAKING_CLASS_ZWJ
 }
+
+fn to_length(value: &LogicalVec2<CSSPixelLength>) -> LogicalVec2<Au> {
+    LogicalVec2 {
+        inline: value.inline.into(),
+        block: value.block.into(),
+    }
+}
+
+fn to_au_rect(value: LogicalRect<Au>) -> LogicalRect<CSSPixelLength> {
```

Comment:
Could these functions be moved into `geom.rs` and implemented as either `From` or `Into` trait implementations? That way they could be shared.